docs/container.md

```diff
@@ -0,0 +1,69 @@
+<!--
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
+-->
+
+# Containerized imageio server
+
+## Overview
+
+ovirt-imageio provides an image for running the ovirt-imageio-daemon
+in a container.
+
+Using a container, users can run an image transfer pod for every
```

Comment:
pod is Kubernetes (k8s) term, but wed don't have any k8s integration yet,
so it is better to talk here a bout a container instead of a pod.